model/WeatherData.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WeatherMonitoringSystem.model
```

Comment:
```suggestion
namespace WeatherMonitoringSystem.Models
```

namespace should be in `PascalCase` and in plural form.